docs/ManualDeploymentGuideScalarDLOnEKS.md

```diff
@@ -1,74 +1,53 @@
 
 ## Step 4. Create a bastion server
 
-For executing some tools to deploy and manage Scalar DL Ledger on EKS, you need to prepare a bastion server in the same VPC of the EKS cluster you created in **Step 3**. Please refer to the following document for more details.  
-
-* [Create a bastion server]() // TODO: Create a new document
+For executing some tools to deploy and manage ScalarDL Ledger on EKS, you need to prepare a bastion server in the same VPC of the EKS cluster you created in **Step 2**. See [Create a bastion server](./CreateBastionServer.md) for more details.
 
 ## Step 5. Prepare a custom values file of Helm 
 
-You need to configure a custom values file for the Helm Chart of Scalar DL Ledger and Scalar DL Schema Loader based on your environment (e.g., access information of the database you created in **Step 2**). Please refer to the following document for more details.  
-
-* [Configure a custom values file of Scalar Helm Chart]() // TODO: Create a new document in the Scalar Helm Chart repository
-
-## Step 6. Deploy Scalar DL Ledger using Scalar Helm Chart
-
-Deploy Scalar DL Ledger on your EKS cluster using Scalar Helm Chart. Please refer to the following document for more details.  
+You need to configure a custom values file for the Helm Chart of ScalarDL Ledger and ScalarDL Schema Loader based on your environment (e.g., access information of the database you created in **Step 3**). See [Configure a custom values file for Scalar Helm Charts](https://github.com/scalar-labs/helm-charts/blob/main/docs/configure-custom-values-file.md) for more details.
 
-* [Deploy Scalar Products using Scalar Helm Chart]() // TODO: Create a new document in the Scalar Helm Chart repository
+## Step 6. Deploy ScalarDL Ledger using Scalar Helm Chart
 
-## Step 7. Check the status of Scalar DL Ledger deployment
+Deploy ScalarDL Ledger on your EKS cluster using Scalar Helm Chart. See [Deploy Scalar products using Scalar Helm Charts](https://github.com/scalar-labs/helm-charts/blob/main/docs/how-to-deploy-scalar-products.md) for more details.
 
-After deploying Scalar DL Ledger on your EKS cluster, you need to check the status of each component. Please refer to the following document for more details.  
+Note: We recommend creating a dedicated namespace by using the `kubectl create ns scalardl-ledger` command and deploying ScalarDL Ledger in the namespace by using the `-n scalardl-ledger` option with the `helm install` command.
 
-* [What you might want to check on a regular basis](./RegularCheck.md) // TODO: Update existing document
+## Step 7. Check the status of ScalarDL Ledger deployment
 
-## Step 8. Monitoring for Scalar DL Ledger deployment
+After deploying ScalarDL Ledger on your EKS cluster, you need to check the status of each component. See [What you might want to check on a regular basis](./RegularCheck.md) for more details.
```

Comment:
I updated the title of the linked doc below to be more specific by referencing the Kubernetes environment. If this title looks OK, let's update the title of the actual `RegularCheck.md` doc as well.

```suggestion
After deploying ScalarDL Ledger in your EKS cluster, you must check the status of each component. For details, see [Components to Regularly Check When Running in a Kubernetes Environment](./RegularCheck.md).
```